massa-models/src/config/constants.rs

```diff
@@ -87,14 +87,22 @@ lazy_static::lazy_static! {
     /// node chain id (to avoid replay attacks)
     pub static ref CHAINID: u64 = {
         // MASM (MainNet):           77658377
-        // MASB (BuildNet):          77658366
         // MASS (SecureNet):         77658383
+        // MASB (BuildNet):          77658366
         // MASL (Labnet):            77658376
         // SANDBOX (Sandbox):        77
-        if cfg!(feature = "sandbox") {
-            77
-        } else {
-            77658383
+        match VERSION.to_string() {
```

Comment:
this check is cleaner: `VERSION.instance == "SAND"`

Reply:
instance is private. Not sure this is a good idea to make it pub ?

Reply:
I think adding a public Version::get_instance_name() function is a good idea:
* prevents accidental mutability that would happen if public
* allows users to not assume anything about the internal or serialized representation of the Version